packages/server/src/main/resources/diagram-resource.ts

```diff
@@ -33,4 +36,17 @@ export class DiagramResource {
       res.status(200).send(token);
     });
   };
+
+  convertSvgToPdf = (req: any, res: any) => {
```

Comment:
Not sure if you have power of this here but it would be nice to have typing here instead of using `any`

Reply:
I added the types.